InstagramKit/Engine/InstagramEngine.h

```diff
@@ -562,31 +528,26 @@ NS_ASSUME_NONNULL_BEGIN
                         withSuccess:(InstagramResponseBlock)success
                             failure:(nullable InstagramFailureBlock)failure;
 
-
 /**
- *  Get the list of users this user follows.
+ *  Get the list of users this self follows.
  *
- *  @param userId   Id of the User object.
  *  @param success  Provides an array of User objects and Pagination info.
  *  @param failure  Provides an error and a server status code.
  */
-- (void)getUsersFollowedByUser:(NSString *)userId
-                   withSuccess:(InstagramUsersBlock)success
+- (void)getUsersFollowedBySelf:(InstagramUsersBlock)success
```

Comment:
Let's stick to Objective C conventions with this. `getUsersFollowedBySelfWithSuccess:`